tutorials/makefile-common

```diff
@@ -2,24 +2,27 @@
 
 # MLIR-AIE install directory. If you have sourced utils/env_setup.sh before
 # running make, the following should work to find the AIE install directory.
-AIE_INSTALL ?= $(dir $(shell which aie-opt))/..
+# AIE_INSTALL ?= $(dir $(shell which aie-opt))/..
 
 # An aarch64 sysroot is required for cross-compiling the host code. The default
 # values of these variables assume that you have built a sysroot by running
 # make in platforms/vck190_bare_prod, but you can use other aarch64 sysroots by
-MLIR_AIE_SYSROOT ?= ${AIE_INSTALL}/../platforms/vck190_bare_prod/aie_platform/sw_comp/sysroots/cortexa72-cortexa53-xilinx-linux
+
+MLIR_AIE_SYSROOT ?= ${VITIS_ROOT}/gnu/aarch64/lin/aarch64-linux/aarch64-xilinx-linux/
 
 # The libstdc++ version that is installed in the sysroot given above. This is
 # used for include and library paths. If you built the sysroot with Vitis
-# 2021.2 and PetaLinux 2021.2, libstdc++ 10.2.0 will be installed. 
-LIBCXX_VERSION ?= 10.2.0
+# 2022.2 and PetaLinux 2022.2, libstdc++ 11.2.0 will be installed. 
+LIBCXX_VERSION ?= 11.2.0
 
 # The following flags are passed to both AI core and host compilation for
 # aiecc.py invocations.
 AIECC_FLAGS += --sysroot=${MLIR_AIE_SYSROOT} --host-target=aarch64-linux-gnu
 
+CHESSCC_FLAGS = -p me -P ${VITIS_AIE_INCLUDE_DIR}
+CHESS_FLAGS = -P ${VITIS_AIE_INCLUDE_DIR}
```

Comment:
Do you intend to make this under user control, so that they could switch to AIE2?   I feel like alot of this duplicates what has to be in other places (like aiecc, lit.cfg.py, and xchesscc_wrapper)

Reply:
This is an intermediate step until we have more complete support for AIE2 in place in the tools you list.